end-to-end-test.sh

```diff
@@ -43,6 +43,7 @@ disabled_collectors=$(cat << COLLECTORS
   time
   timex
   uname
+  processes
```

Comment:
If we have fixtures, I think we can enable this in the end-to-end test.  We only disable here for non-deterministic tests.

Reply:
Yes, but we like to include things in the end-to-end as well.

After you `make`, running `end-to-end-test.sh -u` will update the fixture.

Reply:
Thanks for hint